src/main/java/racingcar/view/InputView.java

```diff
@@ -4,14 +4,14 @@
 
 public class InputView {
 
-    public String[] getCarName() {
+    public static String[] getCarName() {
         Scanner scanner = new Scanner(System.in);
         String userInput = scanner.nextLine();
 
         return userInput.split(",");
     }
 
-    public int getTryCount() {
+    public static int getTryCount() {
         Scanner scanner = new Scanner(System.in);
         int tryCount = scanner.nextInt();
```

Comment:
```suggestion
                return scanner.nextInt();
```

`Scanner scanner = new Scanner(System.in)`은 재사용 가능하도록 해주세요 : )